modules/getting-started/proc-techdocs-configure-odf-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-techdocs-configure-odf-operator_{context}"]
+
+= Making object storage accessible to containers using the Operator
+
+`ObjectBucket` information is stored in the {product-short} `ObjectBucketClaim` config map and secret. Adding the `ObjectBucket` information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+
+* `BUCKET_NAME`
+* `BUCKET_HOST`
+* `BUCKET_PORT`
+* `BUCKET_REGION`
+* `BUCKET_SUBREGION`
+* `AWS_ACCESS_KEY_ID`
+* `AWS_SECRET_ACCESS_KEY`
+
+These variables are then used in the TechDocs plugin configuration.
```

Comment:
Not important for right now, but you might want to look into using [snippets](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets) within modules if you're duplicating things like lists of config options